python/lsst/rapid/analysis/quickLook.py

```diff
@@ -0,0 +1,281 @@
+    """
+    newArgs = {}
+    for attr in ("name", "storageClass", "multiple", "doc", "dimensions", "isCalibration", "deferLoad",
+                 "minimum", "lookupFunction"):
+        if hasattr(connection, attr):
+            newArgs[attr] = getattr(connection, attr)
+    return newArgs
+
+
+class QuickLookTaskConnections(IsrTaskConnections):
+    def __init__(self, *, config=None):
+        """Copy isrTask's connections, changing prereq min values to zero.
+
+        Copy all the connections directly for IsrTask, keeping ccdExposure as
+        required as non-zero, but changing all the other PrerequisiteInputs'
+        minimum values to zero.
+        """
+        super().__init__(config=IsrTask.ConfigClass())  # need a dummy config, isn't used other than for ctor
+        for name, connection in self.allConnections.items():
+            if hasattr(connection, 'minimum'):
+                args = _getArgs(connection)
+                if name != "ccdExposure":  # need one input image always
+                    args['minimum'] = 0
+                newConnection = type(connection)(**args)
+                self.allConnections[name] = newConnection
+                setattr(self, name, newConnection)
+
+        exposure = cT.Output(  # called just "exposure" to mimic isrTask's return struct
+            name="quickLookExp",
+            doc="The quickLook output exposure.",
+            storageClass="ExposureF",
+            dimensions=("instrument", "exposure", "detector"),
+        )
+        self.outputExposure = exposure
+
+
+class QuickLookTaskConfig(pipeBase.PipelineTaskConfig,
+                          pipelineConnections=QuickLookTaskConnections):
+    """Configuration parameters for QuickLookTask."""
+
+    doRepairCosmics = pexConfig.Field(
+        dtype=bool,
+        doc="Interpolate over cosmic rays?",
+        default=True,
+    )
+    doWrite = pexConfig.Field(  # XXX do we want this option? If yes, make it work.
+        dtype=bool,
+        doc="Write out the results?",
+        default=True,
+    )
+
+
+class QuickLookTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
+
+    """Task to automatically perform as much isr as possible.
+
+    Automatically performs as much isr as is possible, depending on the
+    calibration products available. Applies these, and then optionally
+    interpolates over cosmic rays.
+    """
+
+    ConfigClass = QuickLookTaskConfig
+    RunnerClass = pipeBase.ButlerInitializedTaskRunner
+    _DefaultName = "quickLook"
+
+    def __init__(self, *, butler=None, **kwargs):
+        super().__init__(**kwargs)
+
+    def run(self, ccdExposure, *,
+            camera=None,
+            bias=None,
+            dark=None,
+            flat=None,
+            defects=None,
+            linearizer=None,
+            crosstalk=None,
+            bfKernel=None,
+            bfGains=None,
+            ptc=None,
+            crosstalkSources=None,
+            isGen3=False,
```

Comment:
It was only there because the default value in isrTask is currently `False` and hard-coding it to `True` felt wrong, but I'm inclined to agree, and have done so. Once this package is under CI, when that option disappears from the task's run method, we'll easily see that it needs fixing here (if I manage to add tests for this...)